### DISCUSSION TITLE
Parametrisierter für Koordinaten wie bei "hotspot" in WebGIS4 möglich?

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#118] [Q&A]
Unsere Informatik hat im Dokumentenmanagementsystem die Atlas 4 Darstellung über folgende URL realisiert:
http://vogis.cnv.at/atlas/init.aspx?&hotspot=digitaler_atlas_vorarlberg|planung_und_kataster|1:2500|-36286.60|252729.00|hotspot0.gif|
Sie möchten jetzt von mir eine analoge URL für WebGIS5, welche die identische Koordinateneingabe ermöglicht, also  wieder 
"|[Maßstab]|[Rechtswert]|[Hochwert]|[Marker]|".
Man müsse sonst das Dokumentenmanagementsystem anpassen.

Ist es möglich die Parameter in dieser Form einzugeben? (hab's probiert, aber nicht geschafft).

**Comment by gis-eni** [comment]
Ich glaube, da muss man das Dokumentenmanagement sowieso anpassen. Diese Url mit "init.aspx" gibt es in WebGIS nicht mehr.

**Reply by faulgurke** [reply]
Abgesehen davon. Ich verstehe deine Antwort so, dass die Koordinateneingabe in dieser Form nicht möglich ist.

**Reply by gis-eni** [reply]
Genau, es hat sich dazu viel zu viel geändert. Hier konnte man anscheinend auch noch die Karte und irgend eine Darstellung in einem Parameter `hotspot` mitgeben. Das ist und kommt wohl nicht mehr ins WebGIS 5 hinein. Also entweder das Dokumentenmanagement anpassen oder eine `proxy` HTML Seite basteln, die dir diesen Parameter wieder ins neue System weiterleitet.

Du kannst so eine Seite zum Beispiel auch ins `wwwroot` Verzeichnis vom Portal legen (`doc-management-redirect.html`):

``` html

<!DOCTYPE html>
<html>
   <head>
      <title>Datenmanagement Weiterleitung</title>
   </head>
   <body>
      <p>Datenmanagement Weiterleitung</p> </br></br>
	  Webseite funktioniert nicht mít Internet Explorer - Verwenden sie bitte einen anderen Standardbrowser!
     
      <script type="text/javascript">
              const queryString = window.location.search;
              const urlParams = new URLSearchParams(queryString);
             
              let hotspot = urlParams.get('hotspot');
              let hotspotParts = hotspot.split('|');
             
             // Paramter umschreiben in aktuelle Url Parameter
             
             // weiterleiten
              window.location = "https://..."
        </script>
   </body>
</html>

```
